utils/error.hh

```diff
@@ -0,0 +1,92 @@
+/**
+ *		Tempesta FW
+ *
+ * Copyright (C) 2024 Tempesta Technologies, Inc.
+ *
+ * This program is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2 of the License,
+ * or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License along with
+ * this program; if not, write to the Free Software Foundation, Inc., 59
+ * Temple Place - Suite 330, Boston, MA 02111-1307, USA.
+ */
+#pragma once
+
+#include <array>
+#include <cassert>
+#include <cstdio>
+#include <fmt/format.h>
+#include <source_location>
+#include <sstream>
+#include <string>
+#include <stdexcept>
+
+#include <boost/system/system_error.hpp>
+#include <boost/system/error_code.hpp>
+
+class Exception : public std::runtime_error {
+public:
+	Exception(std::string s) noexcept
+		: std::runtime_error(format_syserr(s))
+	{}
+
+	void
+	print(char *buf, size_t len)
```

Comment:
I don't see where this function is used. Also it seems that it is not good to mix c and c++ code (as for me)

Reply:
It was taken from another our project as is. I should look at this file more carefully.
Removed.